src/pages/Profile/Profile.jsx

```diff
@@ -33,12 +54,26 @@ function Profile() {
           <div className="d-flex flex-direction-col gap-1 ">
             <section className="d-flex justify-cont-center ">
               <div className="profile-page-container card-basic profile-card">
-                <div className="d-flex mb-2">
+                <div className="d-flex mb-2 p-relative">
                   <img
-                    className="avatar md"
+                    className="avatar md object-fit-cover"
                     src={user.avatar}
                     alt="user avatar"
                   />
+
+                  {user.username == signInUser && (
+                    <label htmlFor="input-img">
+                      <i className="fa-solid fa-camera select-img-icon"></i>
+                    </label>
+                  )}
+
+                  <input
+                    className="d-none"
+                    type="file"
+                    accept="image/*"
+                    id="input-img"
+                    onChange={(e) => imageHandler(e)}
```

Comment:
here also can directly do `onChange={imageHandler}` event will automatically passed to that function

Reply:
yupp